open-machine-learning-jupyter-book/assignments/prerequisites/python-programming-advanced.ipynb

```diff
@@ -62,46 +62,49 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.    assert is_palindrome(121)==True
</pre><p>add an empty line before this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/324/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>